src/api/ADempiere/common/persistence.js

```diff
@@ -162,6 +164,7 @@ export function getEntity({
 
 /**
  * Object List from window
+ * @deprecated replace with /user-interface/persistence/entities
```

Comment:
Delete it if is deprecate

Reply:
depreciated `getEntities` implementations were removed